src/App.tsx

```diff
@@ -1,25 +1,73 @@
+import { useEffect, useState } from "react";
 import styles from "./App.module.css";
+import { faker } from "@faker-js/faker";
+import FlightTable from "./FlightTable";
+
 
 export interface Flight {
-  // ... fields here
+  flightNumber: string;
+  origin: string;
+  destination: string;
+  departureTime: string;
+  arrivalTime: string;
+  flightDuration: number;
+
 }
 
+
 const App = () => {
-  // state for flights should go here
+
+  const [flightList, setFlightList] = useState([] as Flight[]);
+
+  useEffect(() => {
+    setFlightList(flightList.sort((flightA, flightB) => {
+      return flightA.flightDuration - flightB.flightDuration;
+    }))
```

Comment:
Great job saving code and using the `sort(...)` function inside the `setFlightList(...)` function.